mrtt-ui/src/library/dates/removeExtraTimezoneOffsetFromIsoDate.js

```diff
@@ -0,0 +1,21 @@
+import { DateTime } from 'luxon'
+
+// the MUI date picker assumes we want to display utc time in our
+// local timezone instead, but we do not,
+// so we added some extra time to our server's stored UTC time so when
+// MUI converts it to local time, it actually shows UTC time
+// This function removes the added offset so that the values
+// sent to the server are actually UTC
+// See mrtt-ui/src/library/dates/addExtraTimezoneOffsetToIsoDate.js
+
+export default (isoDateString) => {
+  if (!isoDateString) {
+    return null
+  }
+
+  const luxonUtcDateFromMui = DateTime.fromISO(isoDateString, { zone: 'utc' })
+  const browserTimezoneOffsetMinutes = -DateTime.local().offset
+  const shiftedIsoDateForApi = luxonUtcDateFromMui.minus({ minutes: browserTimezoneOffsetMinutes })
+  const shiftedJustDateNoTimeIsoDateForApi = shiftedIsoDateForApi.toISODate()
```

Comment:
I chose Luxon because its QAed for the math, etc. Also like the `toIsoDate` function to remove the timestamp from the date. Adding time is just false precision, but that could be me being too rigid.

Reply:
k, apparently ISO dates without zones are implicit local time zones of the browser.  Rethinking removing the time component because it looks like you cant add a timezone without a time 

https://sparkgeo.slack.com/archives/CECFDB1QS/p1665172876434679